.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,42 @@
+name: 🐛 Bug
+description: Create a bug report to help us improve
+labels: ['Bug']
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing to iTwinUI by creating an issue!
+        Make sure you first do a search for existing issues so that this is is not a duplicate.
+        Also visit our [FAQ](https://github.com/iTwin/iTwinUI/wiki/FAQ) and fork our [sandbox](https://stackblitz.io/github/iTwin/iTwinUI/tree/main/minimal-sandbox?file=/src/App.tsx)
```

The sandbox link doesn't belong here in the introduction.
```suggestion
        Also visit our [FAQ](https://github.com/iTwin/iTwinUI/wiki/FAQ).
```

---

changes made!